tools/yaml-templates/ios-test.yml

```diff
@@ -18,7 +18,10 @@ steps:
       targetType: inline
       script: |
         brew install xctesthtmlreport
```

Comment:
If this fails (I assume this is the command that is failing intermittently?) is there a way to get more verbose logs about what the failure is?

Reply:
I think they typically hide logs for us when we install this. Basically the error log it throws to console is the detailed one. Not sure if we can get more detailed log on VM.